crates/polars-stream/src/nodes/io_sources/ipc.rs

```diff
@@ -558,20 +569,15 @@ impl MultiScanable for IpcSourceNode {
             }
         }
     }
-    fn with_row_index(&mut self, row_index: Option<PlSmallStr>) {
-        self.row_index = row_index.map(|name| RowIndex { name, offset: 0 });
-    }
 
-    async fn row_count(&mut self) -> PolarsResult<IdxSize> {
+    async fn unrestricted_row_count(&mut self) -> PolarsResult<IdxSize> {
```

Comment:
Can we describe what is meant with this?

Reply:
It is documented in the trait definition.